includes/public/class-wppr-public.php

```diff
@@ -450,8 +459,18 @@ public function show_comment_ratings( $text ) {
 					';
 		}
 		$return .= '</div>';
-
 		return $return . $text . '<div class="cwpr_clearfix"></div>';
 	}
 
+	/**
+	 * AMP support for WPPR
+	 */
+	public function wppr_amp_support() {
+		$output = '';
```

Comment:
@rohittm i'm not sure why you are not enqueue using wp_enqueue_style  ?

Reply:
@selul We cannot add styles in AMP, other than the AMP one. So, I am adding the styles needed for the plugin with the AMP standards. Also, the styles generate colors on the basis of ratings, so we cannot enqueue it directly.

Reply:
got it. i will check this out and get back to you.